lang/cs/Org.Apache.REEF.IO/FileSystem/AzureDataLake/IDataLakeStoreClient.cs

```diff
@@ -31,6 +31,6 @@ internal interface IDataLakeStoreClient
         /// <summary>
         /// Returns the account name for the AdlsClient.
```

Comment:
I would update the comment about the expected format.

Reply:
Updated.